playbook/app/pb_kits/playbook/pb_collapsible/docs/_collapsible_custom_main_rails.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= pb_rails("collapsible", props: { name: "default-example" }) do %>
-    <%= pb_rails("background", props: { background_color: "white", position: "sticky", top: "0", cursor:"pointer", data: {"collapsible-main": "true"} }) do %>
+    <%= pb_rails("flex", props: { position: "sticky", top: "0", cursor: "pointer", data: { "collapsible-main": "true", align: "center", gap: "sm", justify: "between" } }) do %>
```

Comment:
I see you're switching out the background kit for the Flex here, you could always do the same on the React docs to avoid the backgroundColor issue I pointed out above. It'll also always better to have both React and Rails consistent when it comes to docs!